pkg/internal/testchain/address.go

```diff
@@ -0,0 +1,85 @@
+package testchain
+
+import (
+	"github.com/nspcc-dev/neo-go/pkg/crypto/hash"
+	"github.com/nspcc-dev/neo-go/pkg/crypto/keys"
+	"github.com/nspcc-dev/neo-go/pkg/encoding/address"
+	"github.com/nspcc-dev/neo-go/pkg/smartcontract"
+	"github.com/nspcc-dev/neo-go/pkg/util"
+)
+
+// privNetKeys is a list of unencrypted WIFs sorted by public key.
+var privNetKeys = []string{
+	"KzfPUYDC9n2yf4fK5ro4C8KMcdeXtFuEnStycbZgX3GomiUsvX6W",
+	"KzgWE3u3EDp13XPXXuTKZxeJ3Gi8Bsm8f9ijY3ZsCKKRvZUo1Cdn",
+	"KxyjQ8eUa4FHt3Gvioyt1Wz29cTUrE4eTqX3yFSk1YFCsPL8uNsY",
+	"L2oEXKRAAMiPEZukwR5ho2S6SMeQLhcK9mF71ZnF7GvT8dU4Kkgz",
+}
+
+var (
+	// ids maps validators order (by public key sorting) to validators number.
+	// ID is a numeric node identifier in private net.
```

Comment:
I don't think it explains where this id comes from, we really need to mention it here and in public functions using different ordering (standby validators list vs. sorted by key).

Reply:
Fixed.